Cesium3DTilesSelection/src/ImplicitQuadtreeLoader.cpp

```diff
@@ -157,7 +157,8 @@ CesiumAsync::Future<TileLoadResult> requestTileContent(
           CesiumGltfReader::GltfReaderOptions gltfOptions;
           gltfOptions.ktx2TranscodeTargets = ktx2TranscodeTargets;
           gltfOptions.applyTextureTransform = applyTextureTransform;
-          GltfConverterResult result = converter(responseData, gltfOptions);
+          GltfConverterResult result =
+              converter(responseData, gltfOptions, nullptr); // XXX
```

Comment:
Did you leave the XXX here as a reminder for something?

Reply:
See comment about ImplicitOctreeLoader.